### ISSUE TITLE
Is there a difference in the performance of the LibriSpeech dataset between deepspeech.pytorch and deepspeech.torch

### ISSUE THREAD

**Issue by hyzhan** [SeanNaren/deepspeech.pytorch#130]
Is there a difference in the performance of the LibriSpeech dataset between deepspeech.pytorch and deepspeech.torch，I trained 8 epoch on Librispeech-train-other and got 40% WER and 17% CER on Librispeech-test-other. And would you like to tell me the parameter setting of [this link.](https://github.com/SeanNaren/deepspeech.torch/tree/master)
Thank you very much for your deepspeech.pytorch code.

**Comment by dlmacedo** [comment]
https://github.com/SeanNaren/deepspeech.pytorch/issues/4

**Comment by ryanleary** [comment]
There is no batch normalization in this version, though alternatives/ways to implement that are being investigated. I would recommend training with the full training set -- not just the `other` and running for more than 8 epochs. 

I also believe that the character error rates in the torch version may be misreported.
